app/views/layouts/govuk_admin_template.html.erb

```diff
@@ -2,6 +2,7 @@
   environment_style = GovukAdminTemplate.environment_style
   environment_label = GovukAdminTemplate.environment_label
   app_home_path = content_for?(:app_home_path) ? yield(:app_home_path) : root_path
+  app_title = content_for?(:app_title) ? yield(:app_title) : GovukAdminTemplate::Config.app_title
```

Comment:
This change, coupled with the others means that there is no longer a default page title - a change in behaviour. If you neglect to supply `content_for(:app_title)` or set the Config option, I think you'll get an error?

I'm not saying this is necessarily a problem, as long as we recognise the change.